networkx/algorithms/community/tests/test_louvain.py

```diff
@@ -43,6 +43,58 @@ def test_partition():
     assert part == partition
 
 
+def test_directed_partition():
```

Comment:
This should probably be parametrized or split up into two separate tests, but that's a minor organizational nit and not a blocker!

Reply:
I actually wanted to parametrize this function but the edgelists to construct the graphs are too big and I wasn't sure how it would look in terms of styling. Maybe one better way is to create a class?

Also, if we split  the test cases (which I am in favor of) do you think naming the functions something like `test_5175_issue` be good?

Reply:
> but the edgelists to construct the graphs are too big and I wasn't sure how it would look in terms of styling.

I agree - black will likely force this test to be less readable, so I'd say let's not worry about it for now :). There are ways to "trick" black into looking better here (i.e. define the edgelists externally and pass them into `parametrize`), but I'd say that's better for a follow-up PR - the current tests LGTM (minus the missing `G.add_nodes_from`).